web/cypress/e2e/rest/service-edit-service-with-upstream.cy.js

```diff
@@ -73,7 +73,6 @@ context('Edit Service with Upstream', () => {
     cy.get(selector.description).type(data.description);
     cy.get(selector.upstreamSelector).click();
     cy.contains(data.upstreamName).click();
-    cy.get(selector.input).should('be.disabled');
```

Comment:
If you don't click next, some will be banned and some will not. Its positioning of elements is not accurate. It may refer to all input boxes, but there are forbidden and good input boxes. No matter whether it is '.not ' or '.be', it will make mistakes. If it refers to the box just entered, there is something wrong with its writing. It's better to delete it.